src/policy/policy.cpp

```diff
@@ -183,3 +183,9 @@ int32_t Policy::blockSizeAcceptLimit()
         LogPrintf("BlockSize set to extremely low value (%d bytes), this may cause failures.\n", limit);
     return limit;
 }
+
+int64_t Policy::blockSigOpAcceptLimit(int32_t nBlockSize)
+{
+    uint64_t nBlockSizeMb = 1 + (nBlockSize - 1) / 1000000;
```

Comment:
No ceil. It is floating point. You don't want floating point in consensus code as it is non deterministic. If you want to be scared, know that the rounding mode on X86 is controlled by a flag register that isn't saved/restored by the ABI, even when doing round trips to the kernel, and there are war stories out there of driver changing the rounding mode during an IRQ.